src/gc/gc.cpp

```diff
@@ -26685,7 +26683,7 @@ void gc_heap::background_grow_c_mark_list()
         assert (new_c_mark_list);
         memcpy (new_c_mark_list, c_mark_list, c_mark_list_length*sizeof(uint8_t*));
         c_mark_list_length = c_mark_list_length*2;
-        delete c_mark_list;
+        delete [] c_mark_list;
```

Comment:
yeah, I am just pointing out that this is how it's usually done (where there's no dtor to call for elements we delete with objects allocated with new[] with scalar delete). I think this gives a tiny bit of perf improvement. but none of these are created/deleted often so if we are going to change we should change them all to delete[], especially if it does make sense to avoid errors on Linux. it's just kind of odd that you are only changing one of them. I would prefer consistency so perhaps you could open an issue that changes all of the delete's to delete[]'s for objects allocated with new[]?